docs/configuration.md

```diff
@@ -56,30 +56,48 @@ You can think of Quartz plugins as a series of transformations over content.
 
 ![[quartz transform pipeline.png]]
 
-```ts
+```ts title="quartz.config.ts"
 plugins: {
   transformers: [...],
   filters: [...],
   emitters: [...],
 }
 ```
 
-- [[making plugins#Transformers|Transformers]] **map** over content (e.g. parsing frontmatter, generating a description)
-- [[making plugins#Filters|Filters]] **filter** content (e.g. filtering out drafts)
-- [[making plugins#Emitters|Emitters]] **reduce** over content (e.g. creating an RSS feed or pages that list all files with a specific tag)
+- [[tags/plugin/transformer|Transformers]] **map** over content (e.g. parsing frontmatter, generating a description)
+- [[tags/plugin/filter|Filters]] **filter** content (e.g. filtering out drafts)
+- [[tags/plugin/emitter|Emitters]] **reduce** over content (e.g. creating an RSS feed or pages that list all files with a specific tag)
 
-By adding, removing, and reordering plugins from the `tranformers`, `filters`, and `emitters` fields, you can customize the behaviour of Quartz.
+You can customise the behaviour of Quartz by adding, removing and reordering plugins in the `transformers`, `filters` and `emitters` fields.
```

Comment:
Oh I also forgot to mention.

Probably worth saying here is that the order of items in the array matters, atm.

So i.e: FrontMatter should always get set first, Latex second, etc.

@jackyzha0

Reply:
It's right next to this sentence:

![image](https://github.com/jackyzha0/quartz/assets/1475672/588a58dd-bbe9-46d9-9c0c-b7b05b3c911e)

Reply:
oh i didn't catch that. Can you highlight `position sensitive`, or maybe I guess the term is `location-dependent` 😄